package.json

```diff
@@ -136,7 +136,7 @@
     "react-router": "^6.0.0",
     "react-router-dom": "^6.0.0",
     "react-transition-group": "^4.4.2",
-    "recharts": "^2.1.13",
+    "recharts": "2.1.12",
```

Comment:
pinned version to 2.1.12 for now because of this tooltip rendering issue: https://github.com/recharts/recharts/issues/2943